recipe-server/karma.conf.js

```diff
@@ -28,6 +29,10 @@ module.exports = function (config) {
     // The first config is for the control interface. It is only coincidence
     // that this config works for the action code/tests as well.
     webpack: Object.assign(WEBPACK_CONFIG[0], {
+      plugins: WEBPACK_CONFIG[0].plugins.filter(
+        // karma-webpack is not compatible with CommonsChunkPlugin, so exclude it
+        p => !(p instanceof webpack.optimize.CommonsChunkPlugin)
```

Comment:
Yeah, I agree, since this is where we wanted it the most. We can watch this issue to see if it ever gets fixed: https://github.com/webpack-contrib/karma-webpack/issues/22. There may also be a workaround, but I didn't want to dig into it right now.